protocol.html

```diff
@@ -563,61 +563,144 @@ <h3 property="schema:name"><span>Issue</span>: Solid Notification Vocabulary and
 
                       <p>
-                        <span about="" id="resource-server-notification-channel-resource-accept" rel="spec:requirement" resource="#resource-server-notification-channel-resource-accept">
+                        <span about="" id="resource-server-description-resource-accept" rel="spec:requirement" resource="#resource-server-description-resource-accept">
                           <span property="spec:statement">
-                            <span rel="spec:requirementSubject" resource="#ResourceServer">Resource Servers</span>
-                            <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> accept requests targeting the Notification Channel Resource when the value of the <code>Accept</code> header indicates a preferred representation in <code>application/ld+json</code> [<cite><a class="bibref" href="#bib-json-ld11">JSON-LD11</a></cite>].
+                            <a rel="spec:requirementSubject" href="#ResourceServer">Resource Servers</a>
+                            <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> accept requests targeting the <a href="#description-resource">description resource</a> when the value of the <code>Accept</code> header indicates a preferred representation in <code>application/ld+json</code> [<cite><a class="bibref" href="#bib-json-ld11">JSON-LD11</a></cite>].
                           </span>
                         </span>
                       </p>
 
-                      <p>The data model of the notification channel and its capabilities is defined in <cite><a href="#notification-channel-conformance" rel="rdfs:seeAlso">Notification Channel Conformance</a></cite>.</p>
+                      <p>The <a href="#description-resource">description resource</a> has the following properties:</p>
```

Comment:
```suggestion
   <h4 property="schema:name">Description Resource Data Model</h4>
                      <p>The <a href="#description-resource">description resource</a> has the following properties:</p>
```

Or without *Data Model* suffix if we reconcile it with the term definition.
This change would also require a `<section>` with `id` around it